src/wp-includes/media.php

```diff
@@ -343,9 +346,10 @@ function remove_image_size( $name ) {
  * @param int        $height Image height in pixels.
  * @param bool|array $crop   Optional. Whether to crop images to specified width and height or resize.
  *                           An array can specify positioning of the crop area. Default false.
+ * @param bool       $output_mimes Whether to output secondary mimes for the post thumbnail. Default is true.
  */
-function set_post_thumbnail_size( $width = 0, $height = 0, $crop = false ) {
-	add_image_size( 'post-thumbnail', $width, $height, $crop );
+function set_post_thumbnail_size( $width = 0, $height = 0, $crop = false, $output_mimes = true ) {
+	add_image_size( 'post-thumbnail', $width, $height, $crop, true );
```

Comment:
It's a bit odd to have the parameter here, but also have `post-thumbnail` hard-coded in the list of core sizes above. Maybe we should omit this parameter on the function here for now and always set it to `true` inside? It's a bit odd to have it on `set_post_thumbnail_size()` when it has a different default than on `add_image_size()`.

Reply:
Ok, I can remove for now and we can consider later. main idea was to add a way to control the output when calling set_post_thumbnail_size but agree it feels odd.

Reply:
removed